common/src/main/scala/com/intuit/data/simplan/common/github/CommitServiceExtention.scala

```diff
@@ -0,0 +1,32 @@
+package com.intuit.data.simplan.common.github
+
+import com.google.gson.reflect.TypeToken
+import org.eclipse.egit.github.core.{IRepositoryIdProvider, RepositoryCommit}
+import org.eclipse.egit.github.core.client.{GitHubClient, PageIterator, PagedRequest}
+import org.eclipse.egit.github.core.service.CommitService
+
+import java.util
+import java.util.List
+
+class CommitServiceExtention (client: GitHubClient) extends CommitService (client)  {
```

Comment:
Do we need to create a separate class for this? Cant this method be added as just an additional method in `GithubInteractionHandler`

Reply:
yea i believe a separate class is needed because otherwise i'd need to make a call via an endpoint with parameters directly, which would be more string parsing, and the `this.___` calls are not available outside of CommitService